### PR TITLE
cross-mio updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Added GI gliders 538 and 575 to vocab.csv (first Irminger deployment for both gliders).
Added glider CTD sensor to sensor_bulk_load-AssetRecord.csv (CGINS-CTDGVM-09236).  This is a new sensor for glider on upcoming Papa 9a glider cruise (for glider 363 --> FCP #3304-00160-00060).

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[Irminger 9 AM Updates [DT-160]](https://trello.com/c/YNyU7qml/160-irminger-9-am-updates-dt-160)

**Comment by rtravis-whoi** [comment]
[Papa 9 AM Updates [DT-161]](https://trello.com/c/n2lTwbTF/161-papa-9-am-updates-dt-161)

**Review by cdobs — APPROVED** [review]
Looks good

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 1]
```diff
@@ -662,6 +662,7 @@ CGINS-CTDGVM-09230,N00650,Sensor,1,CTD for Glider: CTDGV Series M,CTDGV for Sloc
 CGINS-CTDGVM-09231,N00653,Sensor,1,CTD for Glider: CTDGV Series M,CTDGV for Slocum Glider,Sea-Bird Electronics,SBE Slocum Glider Payload CTD (GP-CTD),9231,,20141223,,,
 CGINS-CTDGVM-09232,N00644,Sensor,1,CTD for Glider: CTDGV Series M,CTDGV for Slocum Glider,Sea-Bird Electronics,SBE Slocum Glider Payload CTD (GP-CTD),9232,,20150326,,,
 CGINS-CTDGVM-09235,N00641,Sensor,1,CTD for Glider: CTDGV Series M,CTDGV for Slocum Glider,Sea-Bird Electronics,SBE Slocum Glider Payload CTD (GP-CTD),9235,,20150316,171195,,
+CGINS-CTDGVM-09236,,Sensor,1,CTD for Glider: CTDGV Series M,CTDGV for Slocum Glider,Sea-Bird Electronics,SBE Slocum Glider Payload CTD (GP-CTD),9236,,,,,
```
[the remark this answers is not in the document]
Comment (snwhite) [reply bulk/sensor_bulk_load-AssetRecord.csv:665]: Looks good.

File: vocab/vocab.csv [hunk 2]
```diff
@@ -1205,6 +1205,11 @@ GI05MOAS-GL537-00-ENG000000,Global Irminger Sea,Mobile Assets,Open Ocean Glider
 GI05MOAS-GL537-01-FLORDM000,Global Irminger Sea,Mobile Assets,Open Ocean Glider 537,2-Wavelength Fluorometer,WET Labs,ECO Puck FLBB-SLC,0,1000
 GI05MOAS-GL537-02-DOSTAM000,Global Irminger Sea,Mobile Assets,Open Ocean Glider 537,Dissolved Oxygen,Aanderaa,Optode 4831,0,1000
 GI05MOAS-GL537-04-CTDGVM000,Global Irminger Sea,Mobile Assets,Open Ocean Glider 537,CTD,Sea-Bird,SBE Glider Payload CTD (GP-CTD),0,1000
+GI05MOAS-GL538,Global Irminger Sea,Mobile Assets,Open Ocean Glider 538,,,,0,1000
+GI05MOAS-GL538-00-ENG000000,Global Irminger Sea,Mobile Assets,Open Ocean Glider 538,Mobile Asset Controller,Teledyne Webb,G2 Slocum Gliders,0,1000
+GI05MOAS-GL538-01-FLORDM000,Global Irminger Sea,Mobile Assets,Open Ocean Glider 538,2-Wavelength Fluorometer,WET Labs,ECO Puck FLBB-SLC,0,1000
+GI05MOAS-GL538-02-DOSTAM000,Global Irminger Sea,Mobile Assets,Open Ocean Glider 538,Dissolved Oxygen,Aanderaa,Optode 4831,0,1000
+GI05MOAS-GL538-04-CTDGVM000,Global Irminger Sea,Mobile Assets,Open Ocean Glider 538,CTD,Sea-Bird,SBE Glider Payload CTD (GP-CTD),0,1000
```
[the remark this answers is not in the document]
Comment (snwhite) [reply vocab/vocab.csv:1212]: Looks good.

File: vocab/vocab.csv [hunk 3]
```diff
@@ -1247,6 +1252,14 @@ GI05MOAS-PG565-03-FLORTM000,Global Irminger Sea,Mobile Assets,Profiling Glider 5
...
+GI05MOAS-PG575,Global Irminger Sea,Mobile Assets,Profiling Glider 575,,,,0,200
+GI05MOAS-PG575-00-ENG000000,Global Irminger Sea,Mobile Assets,Profiling Glider 575,Mobile Asset Controller,Teledyne Webb,G2 Slocum Gliders,0,200
+GI05MOAS-PG575-01-CTDGVM000,Global Irminger Sea,Mobile Assets,Profiling Glider 575,CTD,Sea-Bird,SBE Glider Payload CTD (GP-CTD),0,200
+GI05MOAS-PG575-02-DOSTAM000,Global Irminger Sea,Mobile Assets,Profiling Glider 575,Dissolved Oxygen,Aanderaa,Optode 4831,0,200
+GI05MOAS-PG575-03-FLORTM000,Global Irminger Sea,Mobile Assets,Profiling Glider 575,3-Wavelength Fluorometer,WET Labs,ECO Puck FLBBCD-SLK,0,200
+GI05MOAS-PG575-04-FLORTO000,Global Irminger Sea,Mobile Assets,Profiling Glider 575,3-Wavelength Fluorometer,WET Labs,ECO BB3,0,200
+GI05MOAS-PG575-05-NUTNRM000,Global Irminger Sea,Mobile Assets,Profiling Glider 575,Nitrate,Satlantic,SUNA V2,0,200
+GI05MOAS-PG575-06-PARADM000,Global Irminger Sea,Mobile Assets,Profiling Glider 575,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,200
```
[the remark this answers is not in the document]
Comment (snwhite) [reply vocab/vocab.csv:1262]: Looks good.

**Review by snwhite — APPROVED** [review]

**Pull request merged**